### PR TITLE
Update compose.tpl

### PR DESCRIPTION
**Pull request by loma-one**
The change ensures that the input dialogue is expanded when the bottom line is reached.
https://github.com/friendica/friendica/issues/14339

### CODE REVIEW

**Comment by loma-one** [comment]
https://github.com/friendica/friendica/issues/14339

**Comment by haheute** [comment]
I have also looked at textareas today, but in the edit event modal. There is a javascript active, called [autosize](https://github.com/friendica/friendica/tree/2024.06-rc/view/theme/frio/frameworks/autosize). The class for it is `text-autosize` if I remember correctly. Saw this class here, too... wonder if `autosize` should work here.

The autosize script seems to be an older version, but I couldn't find out where this could be updated.

File: view/templates/item/compose.tpl [hunk 1]
```diff
@@ -103,4 +103,15 @@
 </div>
 <script>
 	dzFactory.setupDropzone('#dropzone-{{$id}}', 'comment-edit-text-{{$id}}');
+		document.addEventListener("DOMContentLoaded", function() {
```
Comment (annando) [inline view/templates/item/compose.tpl]: Can you fix the indentation?

Comment (loma-one) [reply view/templates/item/compose.tpl]: Corrected indentation

File: view/templates/item/compose.tpl [hunk 2]
```diff
@@ -103,4 +103,19 @@
...
+		textarea.addEventListener("input", function() {
+			this.style.height = "auto";
+			this.style.height = (this.scrollHeight) + "px";
+		});
+
+		// Set initial height
+		textarea.style.height = "auto";
+		textarea.style.height = (textarea.scrollHeight) + "px";
+	});
+});
```
Comment (annando) [inline view/templates/item/compose.tpl]: Sorry for nitpicking, but this closing bracket doesn't align to the opening one, which make the code harder to read.

Comment (loma-one) [reply view/templates/item/compose.tpl]: I'm not sure if I've understood you correctly. I'm still practising at the moment and would be grateful for any tips

**Pull request merged**